src/World.cpp

```diff
@@ -1042,9 +1044,60 @@ void cWorld::Tick(std::chrono::milliseconds a_Dt, std::chrono::milliseconds a_La
 		SaveAllChunks();
 	}
 
-	if (m_WorldAge - m_LastUnload > std::chrono::seconds(10))  // Unload every 10 seconds
+
+	// See if we should unload chunks. Cuberite likes to keep as many chunks as possible in RAM for maximum performance.
+	// Chunks are unloaded only when the allocated chunk space (configurable in settings.ini) is full.
+	if (m_WorldAge - m_LastUnload > std::chrono::seconds(2))
 	{
-		UnloadUnusedChunks();
+		cRoot * Root = cRoot::Get();
+		int TotalChunks = Root->GetTotalChunkCount();           // The amount of loaded chunks in all worlds combined
+		int MyChunks = GetNumChunks();                          // Our amount of loaded chunks
```

Comment:
The chunk count is probably not the best measure of chunk memory usage due to varying chunk sizes, but is probably the simplest way of getting an approximate measure. And its clear that that is what the setting is doing, so I think this is a good first implementation.